src/utils/indexForSearch.ts

```diff
@@ -53,28 +71,49 @@ export const indexForSearch = async () => {
           image_url = resp.data.marketingCollection.artworks.hits[0].imageUrl
         } catch (error) {
           console.log(
-            `${collection.slug} - error fetching artworks: `,
+            `\t[error] - ${collection.slug} - error fetching artworks: `,
             error.message
           )
+          bogus_collections.push({
+            slug: collection.slug,
+            reason: "error fetching artworks",
+            data: error.message,
+          })
+          continue
```

Comment:
This is the root cause of the issues we've been seeing where artworks with no images show up in prod data - previously, the code had anticipated this error and logged it out _but it never called `continue`_ so the code continued as if image_url had been set.